apps/ledger-live-desktop/src/internal/index.ts

```diff
@@ -52,13 +55,14 @@ if (INITIAL_SENTRY_TAGS) {
   if (parsed) setTags(parsed);
 }
 
-process.on("message", (m: Message) => {
+// Handles messages from the `main` thread
+process.on("message", async (m: Message) => {
```

Comment:
does process.on do anything special with the promise? i'm not sure it do

Reply:
I've set it as `async` because some of the message handlers are async and needs to be awaited - was it your question ? Or are you talking a promise that would be returned from `process.on` ?

Reply:
I'm actually refactoring this part again, so the `async` will disappear: 

I'm going to use Observable, and remove the `process.send` from `transportHandler` to decouple the transport handler logic and how the response is sent back to the main process -> this structure allows to easily unit test the `transportHandler` ;)

Reply:
Shouldn't we remove the `async` keyword now ?

Reply:
I'm dumb, i re-wrote the handlers with Observable but forgot to remove the `async` keyword, thx !

Reply:
done